app/javascript/menu/side-nav-menu-link.jsx

```diff
@@ -27,7 +29,15 @@ const SideNavMenuLink = ({
       <Link className={className} onClick={onClick} onKeyPress={onClick} tabIndex="0">
         {IconElement && (
           <SideNavIcon small>
-            <IconElement />
+            {expanded && (<IconElement />)}
```

Comment:
I don't think we need to overcomplicate this, what about having the tooltip even if the menu is not collapsed?

Reply:
There are two issues by doing that.

- Screen reader will reader both tooltip and text beside it, then it will be like example: Network Network , which seems confusing experience to users 
- when we tab into left nav, tab focus go to icon first as it has tooltip  and then anchor tag, we tab through leftnav for navigation, not to read tooltips, when text is already present, tooltip adding with same text feels completely redundant annoying experience for keyboard users.

<img width="349" alt="Screen Shot 2020-10-12 at 9 10 30 AM" src="https://user-images.githubusercontent.com/37085529/95750647-7215d700-0c6b-11eb-91f5-f605ecb8fcbf.png">